task5/src/main/java/com/example/demo1/name.java

```diff
@@ -0,0 +1,23 @@
+package com.example.demo1;
+
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.*;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(value = "/get")
+public class name extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html");
+        String name = request.getParameter("name");
+        if(name.length() > 100){
+            throw new IOException();
```

Comment:
Always add an exception message describing what exactly happened. Just try to work out  this habit, because it will help you greatly when working in a commercial project